pom.xml

```diff
@@ -112,81 +164,252 @@
+                            <logStdout>true</logStdout>
+                            <images>
+                                <image>
+                                    <name>${docker.repository}:${docker.version-tag}</name>
+                                </image>
+                            </images>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>docker-start</id>
+            <build>
+                <defaultGoal>docker:start</defaultGoal>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>0.33.0</version>
+                        <configuration>
+                            <images>
+                                <image>
+                                    <name>${docker.repository}:${docker.version-tag}</name>
+                                    <alias>${fabric8.namespace}_${project.artifactId}</alias>
+                                    <run>
+                                        <namingStrategy>alias</namingStrategy>
+                                        <dependsOn>
+                                            <container>mongo</container>
+                                        </dependsOn>
+                                        <links>
+                                            <link>mongo</link>
+                                        </links>
+                                        <ports>
+                                            <port>${tcp.port.exposed}:${tcp.port.internal}</port>
+                                        </ports>
+                                        <env>
+                                            <JAVA_OPTIONS>-Dspring.data.mongodb.host=mongo
+                                                -Dtomcat.protocol.ajp=${container.enable-tomcat-ajp}
+                                            </JAVA_OPTIONS>
+                                        </env>
+                                    </run>
+                                </image>
+                            </images>
+                        </configuration>
+
+                        <executions>
+                            <execution>
+                                <id>start</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <goal>stop</goal>
+                                    <goal>build</goal>
+                                    <goal>start</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>asciidoc-test</id>
```

Comment:
-test ?

Reply:
Like Generated ascii docs and test
I can change it to generate-asciidocs-and-test (and hope I've never have to type it :)

Reply:
But it's the default profile, I could also name it "default"

Reply:
Well don't really care. Could be 2 seperate profiles, but asciidoc-test reads like you are testing the asciidocs plugin

Reply:
-> default